sys/include/net/gnrc/lwmac/lwmac.h

```diff
@@ -8,10 +8,60 @@
  */
 
 /**
- * @defgroup    net_lwmac Simplest possible MAC layer
+ * @defgroup    net_gnrc_lwmac Simplest possible MAC layer
  * @ingroup     net_gnrc
  * @brief       Lightweight MAC protocol that allows for duty cycling to save
  *              energy.
+ *
+ * LWMAC implementation
+ * ====================
+ *
+ * # Radio duty cycling
```

Comment:
Here and headings below need to start with `##`, since

```markdown
Heading
=======
```

and

```markdown
# Heading
```

are both level 1 headings.